draft-ietf-ppm-dap.md

```diff
@@ -2367,10 +2288,6 @@ For fixed_size tasks, the batch boundaries are defined by opaque batch IDs. Thus
 the Aggregator needs to check that the query is associated with a known batch
 ID:
```

Comment:
nit: Consider merging the text after the only remaining bullet into this paragraph.